pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -771,44 +761,43 @@ func makeISORootPath(p string) string {
 	return fmt.Sprintf("file://%s", fullpath)
 }
 
-func makeKickstartSudoersPost(names []string) string {
+func makeKickstartSudoersPost(names []string) *osbuild.PostOptions {
 	if len(names) == 0 {
-		return ""
+		return nil
 	}
-	echoLineFmt := `echo -e "%[1]s\tALL=(ALL)\tNOPASSWD: ALL" > "/etc/sudoers.d/%[1]s"
-chmod 0440 /etc/sudoers.d/%[1]s`
+	echoLineFmt := `echo -e "%[1]s\tALL=(ALL)\tNOPASSWD: ALL" > "/etc/sudoers.d/%[1]s"`
+	chmodLineFmt := `chmod 0440 /etc/sudoers.d/%[1]s`
 
 	filenames := make(map[string]bool)
 	sort.Strings(names)
-	entries := make([]string, 0, len(names))
+	post := &osbuild.PostOptions{}
 	for _, name := range names {
 		if filenames[name] {
 			continue
 		}
-		entries = append(entries, fmt.Sprintf(echoLineFmt, name))
+		post.Commands = append(post.Commands,
```

Comment:
Similar to above, this would be easier for me to read if the script was just embedded in the `Sprintf`.